relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -314,15 +316,42 @@ pub fn extract_tags(
                 } else {
                     scrubbed
                 };
-                Url::parse(url).ok().and_then(|url| {
+                if let Some(domain) = Url::parse(url).ok().and_then(|url| {
                     url.domain().map(|d| {
                         let mut domain = d.to_lowercase();
                         if let Some(port) = url.port() {
                             domain = format!("{domain}:{port}");
                         }
                         domain
                     })
-                })
+                }) {
+                    Some(domain)
+                } else if let Some(server_host) = span
+                    .data
+                    .value()
+                    .and_then(|data| data.get("server.address"))
+                    .and_then(|value| value.as_str())
+                {
+                    let lowercase_host = server_host.to_lowercase();
+                    let mut parts = lowercase_host.split(':');
+
+                    let domain = parts.next().unwrap_or("");
+                    let port = parts.next().and_then(|s| s.parse::<u16>().ok());
+                    if let Some(url_scheme) = span
+                        .data
+                        .value()
+                        .and_then(|data| data.get("url.scheme"))
+                        .and_then(|value| value.as_str())
+                    {
+                        span_tags.insert(
+                            SpanTagKey::RawDomain,
+                            format!("{}://{}", url_scheme, domain),
```

Comment:
Shouldn't this include the port if there was one?

Reply:
Good catch!, yeah it should include port. We can just do
```suggestion
                            format!("{}://{}", url_scheme, server_host),
```

Reply:
I assume you';d want `lowercase_host`?

Reply:
yeah good catch!, I updated the code to use `lowercase_host`